packages/components/tree/src/model/tree-store.ts

```diff
@@ -388,7 +389,7 @@ export default class TreeStore {
   }
 
   setCurrentNodeKey(key?: TreeKey, shouldAutoExpandParent = true): void {
-    if (key === null || isUndefined(key)) {
+    if (isNil(key) || isUndefined(key)) {
```

Comment:
@Simon-He95 if you use `isNil`, you can get rid of `isUndefined` here, since `isNil` includes both `null` and `undefined` here.